web/packages/api/src/assets_v2.ts

```diff
@@ -95,7 +99,7 @@ export type Source = {
     type: SourceType
     id: string
     key: string
-    destinations: { [destination: string]: string[] }
+    destinations: { [destination: string]: { type: SourceType; assets: string[] } }
```

Comment:
Since this changes the expected data structure, we would mark this release as a major version bump, right?